src/components/forms/PlanForm/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -68,7 +68,7 @@ exports[`containers/forms/PlanForm - Edit ACTIVE renders all fields correctly in
   readOnly={false}
```

Comment:
@justinlewis why does this PR result in changes to this file?  It should not be touching this code, no?